final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,143 @@
+import feedparser
+import sys
+import datetime
+import argparse
+import json
+from bs4 import BeautifulSoup
+
+DEFAULT_LIMIT_COUNT = 3
+VERSION = '1.0'
+
+
+class RSSReader:
+    is_verbose = False  # logging is disabled by default
+    is_json = False
+    limit = DEFAULT_LIMIT_COUNT
```

Comment:
If you are going to treat the whole reader as a class, it is better to make these instance attributes, instead of class attributes.